config/storage.yml

```diff
@@ -5,3 +5,10 @@ test:
 local:
   service: Disk
   root: <%= Rails.root.join("storage") %>
+
+s3:
+  service: S3
+  region: ap-southeast-2
+  access_key_id: <%= ENV["RAILS_ASSETS_ACCESS_KEY"] %>
+  secret_access_key: <%= ENV["RAILS_ASSETS_SECRET_KEY"] %>
+  bucket: <%= ENV.fetch("RAILS_ASSETS_BUCKET_ID", "nfsa-staging-assets") %>
```

Comment:
not sure about having a default name?
maybe just default to nil for everything? or is bucket name required to boot?

Reply:
oops, yeah good call